<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601916969803' stop='1601917022444'>
    <name>Add KNOS Relationship</name>
    <title>Add KNOS Relationship</title>
    <test-cases>
        <test-case start='1601916969804' status='passed' stop='1601917010502'>
            <name>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</name>
            <title>CancelKNOS- It should cancel adding new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='9f8a55f2-a383-4b27-9da2-a6423e6d466a-attachment.png' type='image/png' size='51619'/>
            </attachments>
        </test-case>
        <test-case start='1601917010502' status='passed' stop='1601917013778'>
            <name>AddNewKNOS- It should add new casetype and subtype releationship to system!</name>
            <title>AddNewKNOS- It should add new casetype and subtype releationship to system!</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='79ea54b7-58ea-4848-b6f3-23c14f2bc5f9-attachment.png' type='image/png' size='49426'/>
            </attachments>
        </test-case>
        <test-case start='1601917013778' status='failed' stop='1601917022444'>
            <name>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</name>
            <title>Verify KNOS- Verifying relationship in Associate KNOS Codes drop down</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d716f86c-a37a-4444-b8b2-72c24b42cc69-attachment.png' type='image/png' size='57892'/>
            </attachments>
            <failure>
                <message>Expected 'Professional Tort' to equal 'CaseType123'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (C:\Users\UX013810\workspace\WCWRUI\Tests\Admin\UC_AD_16\AddNewKNOSTest.js:71:54)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2974:25
                    at C:\Users\X013810\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>